Proteomics/Fragmentation/DissociationTypeCollection.cs

```diff
@@ -41,19 +65,31 @@ public class DissociationTypeCollection
             { ProductType.D, null},// diagnostic ions are not shifted but added sumarily
         };
 
-        //From http://www.matrixscience.com/help/fragmentation_help.html
-        //Low Energy CID -- In low energy CID(i.e.collision induced dissociation in a triple quadrupole or an ion trap) a peptide carrying a positive charge fragments mainly along its backbone, 
-        //generating predominantly b and y ions. In addition, for fragments containing RKNQ, peaks are seen for ions that have lost ammonia (-17 Da) denoted a*, b* and y*. For fragments containing 
-        //STED, loss of water(-18 Da) is denoted a°, b° and y°. Satellite ions from side chain cleavage are not observed.
-        public static Dictionary<ProductType, List<char>> ProductTypeAminoAcidSpecificites = new Dictionary<ProductType, List<char>>
+        private static Dictionary<DissociationType, (double[], double[])> DissociationTypeToNTermMassCaps = new Dictionary<DissociationType, (double[], double[])>();
```

Comment:
I can't figure out what a "mass cap" is.

Reply:
I can change this name, I suppose the current nomenclature is "mass shift"